### ISSUE TITLE
Button name is not announced by screen reader when keyboard lands on button inside Add task blade.

### ISSUE THREAD

**Issue by msftedad** [quilljs/quill#3848]
Button name is not announced by screen reader when keyboard lands on button inside Add task blade.

**Steps for Reproduction**

1. Open https://codepen.io/alexkrolick/pen/xgyOXQ
2. Navigate to the header section inside Add task blade.
3. Turn on Narrator
4. Observe the issue whether the button name is announced by screen reader when keyboard lands on button inside Add task blade or not.

**Expected behavior**:
The Name of the respective button should be announced by the screen reader when keyboard focus moves to button.

**Actual behavior**:
Screen reader does not announce name of the button available under Incident tasks blade.
The Narrator announces as "Button".

**Platforms**:
OS: Win11 Version 22H2 (OS Build 25284.1010)
Browser: Edge Version 1111.0.1633.0(Official build) (64-bit)

Attachments:
https://github.com/quilljs/quill/assets/93735775/e72b6561-b445-4683-ac54-5c057b82d839

![codesnippet-MAS4 1 2-Incident tasks](https://github.com/quilljs/quill/assets/93735775/0f859751-cc72-4ecf-ade6-23ca6d3ff124)

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by luin** [comment]
The recommended way is to provide your own HTML template for toolbars to fully customize toolbar. [Details](https://v2.quilljs.com/docs/modules/toolbar#:~:text=For%20use%20cases%20requiring%20even%20more%20customization)
